app/views/returns/NowStartYourReturnView.scala.html

```diff
@@ -0,0 +1,39 @@
+@*
+ * Copyright 2022 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@import components.{paragraphBody, LinkButton}
+
+@this(
+        layout: templates.Layout,
+        paragraph: paragraphBody,
+        linkButton: LinkButton,
+)
+
+@(obligationPeriod:String, buttonLink: Call)(implicit request: Request[_], messages: Messages)
+
+@layout(pageTitle = titleNoForm(messages("nowStartYourReturn.heading"))) {
```

Comment:
Should the title contain "Submit return" test? E.g:
Now start your return for {July to September 2022} - Submit return - Plastic Packaging Tax - GOV.UK